### ISSUE TITLE
Convert tutorials to tasks

### ISSUE THREAD

**Issue by dwelsch-esi** [etcd-io/website#765]
## Overview

Rename the "Tutorials" section "Tasks". Split this Task section by user role and put the resulting two sections in their respective user guides: Developer and Operations.

Rename each task by removing "How to" and using the "-ing" verb form. Make steps in each task explicit; one-step tasks are okay. 

Rewrite each task as step-by-step instructions for a single task in an individual description. The goal is to walk the user through the completion of the task. For example, the [How to access etcd][access-tutorial] tutorial should be two separate tasks: "Writing to etcd on the CLI" and "Reading from etcd on the CLI".

Audience: The user role to which the task most applies, probably either Developer or Operator.

Type: Task

The existing documentation might contain helpful source material that you can pull into this doc change. See recommendations for the existing (at the time of the CNCF tech doc analysis) etcd technical documentation pages:
https://github.com/cncf/techdocs/blob/main/assessments/0010-etcd/etcd-implementation.md/#general-reorganization

## 🎤 Context

This issue tracks recommended changes resulting from an analysis of the etcd documentation commissioned by CNCF. The analysis and supporting documents are here: https://github.com/cncf/techdocs/tree/main/assessments/0010-etcd/

## ✌️ Possible Implementation

Related material in the current doc:
- https://github.com/etcd-io/website/tree/main/content/en/docs/v3.5/tutorials

Suggested changes:

Use the following outline for each task:

- Prerequisites (bullet list of prerequisite conditions, if any)
- Procedure
    1. Step 1 (keep steps discrete and atomic. Put command-line input and expected output in a code block.)
    2. Step 2 ...
- Result (optional; describe output or side effects if they're notable or unexpected.)

**Comment by celestehorgan** [comment]
In preparation for the Contributor Fest docs hackathon, I've created the following sub-issues. @nate-double-u  you can leave these off the issue tracker until the end of KubeCon EU, I think :) 


- https://github.com/etcd-io/website/issues/790 
- https://github.com/etcd-io/website/issues/791
- https://github.com/etcd-io/website/issues/792
- https://github.com/etcd-io/website/issues/793 (note for the contributor summit: this one is a little more involved and would do well for a pair of contributors!)
- https://github.com/etcd-io/website/issues/794 (Probably requires help from @wenjiaswe or another etcd contributor to call it complete)
- https://github.com/etcd-io/website/issues/795
- https://github.com/etcd-io/website/issues/796
- https://github.com/etcd-io/website/issues/797
- https://github.com/etcd-io/website/issues/798
- https://github.com/etcd-io/website/issues/799
- https://github.com/etcd-io/website/issues/800 (This one is a little more complex!) 
- https://github.com/etcd-io/website/issues/801
- https://github.com/etcd-io/website/issues/802

**Comment by wendy-ha18** [comment]
Just want to document the discussion about this issue from etcd-doc mentoring meeting today 12 Jun:

If we want to reorganising the directories from current:
```
docs/
└── v3.6/
    ├── tutorials/
    │   ├── _index.md
    │   ├── doc1.md
    │   ├── doc2.md
```

 to rename `tutorials` to `tasks` and splitting it into `developer` and `operator` sections.
```
docs/
└── v3.6/
    ├── tasks/                     # ← renamed from tutorials
    │   ├── developer/
    │   │   ├── _index.md    
    │   │   └── doc1.md            # moved from tutorials/
    │   └── operator/
    │       ├── _index.md
    │       └── doc2.md.          # moved from tutorials/
```

We should think about how we handle redirects from old URLs to new ones before we do it. 

@jberkus mentioned that changing file paths will also change URLs, which can break links that users might find on Google. For example, if someone searches for "How to write to etcd" and the page URL has changed, the old link won't work anymore, which can effect our search ranking. If etcd website lose visibility/ranking on Google, users may end up on outdated or incorrect pages from other sources like Stack overflows or Blog posts instead of ours.

@nate-double-u do you have any suggestion how we set up redirects URLs or any other ideas arround this issue?

cc @ronaldngounou

**Comment by nate-double-u** [comment]
> [@nate-double-u](https://github.com/nate-double-u) do you have any suggestion how we set up redirects URLs or any other ideas arround this issue?

Yes, we can use the [index.redirects](https://github.com/etcd-io/website/blob/main/layouts/index.redirects) file to manage any redirects we want

**Comment by ivanvc** [comment]
With #1029 merged (btw, awesome job, @wendy-ha18 ;), I think we can close this issue, right?

**Comment by jberkus** [comment]
Yes!
